src/components/layouts/Navigation.tsx

```diff
@@ -7,11 +7,32 @@ export const Navigation = () => {
     <Header>
       <Navigator>
         <Link href="/"><img src="/assets/logo.png" alt="logo" /></Link> 
-        <ul>
-          <li><Link href="/about">about</Link></li>
-          <li><Link href="/products">products</Link></li>
-          <li><Link href="/events">events</Link></li>
-        </ul> 
+        <ProductList>
+          <li className="navigation-list"><Link href="/about">about</Link></li>
```

Comment:
p2:코드 통일성이 필요해보입니다. prettier 셋팅 다시한번 확인해주세요
```suggestion
          <li className="navigation-list">
            <Link href="/about">about</Link>
          </li>
```